cmd/skopeo/utils.go

```diff
@@ -244,6 +245,7 @@ func (opts *imageOptions) newSystemContext() (*types.SystemContext, error) {
 }
 
 // imageDestOptions is a superset of imageOptions specialized for image destinations.
+// Every user should call imageDestOptions.warnAboutIneffectiveOptions() as part of handling the CLI
```

Comment:
Care to add this to `sync.go` as well?

Or if you like, we can do that afterwards.

Reply:
@mtrmac Yeah ofc, i'm adding it right now